vendor/github.com/apache/pulsar-client-go/examples/consumer-listener/consumer-listener.go

```diff
@@ -18,47 +18,47 @@
 package main
 
 import (
-    "fmt"
-    "log"
+	"fmt"
```

Comment:
It seems that there are many format changes. Can we do the format changes in a separate pull request?